OffchainLabs/arbitrum-token-bridge
# feat: add posthog events

Closes https://github.com/OffchainLabs/arbitrum-token-bridge/issues/748

This feature aims to batch different analytics providers into a single call in our UI. Events are mapped based on the event format of each provider. E.g. Posthog accepts metadata, Fathom doesn't.

Test steps:
You can see the list of all events in `AnalyticsUtils.ts`. Based on that you can see what events should be sent to Posthog and Fathom (you can see both in the console while in dev).

**Please do not use Posthog API key from Vercel for local testing. If you don't want to [set up an account](https://eu.posthog.com/signup) let me know, I will give you mine.**
